.github/workflows/images.yaml

```
@@ -2,7 +2,7 @@ name: Build images
 
 on:
   push:
-    branches: [ "main", "impl-tgbot" ]
+    branches: [ "main", "impl-tgbot", "new-deploy" ]
```

Давай вот это уберем?

```suggestion
    branches: [ "main" ]
```

---

это уберем перед мерджем, спасибо, что подсветил
оставил, потому что тещу из своей ветки)